next.config.js

```diff
@@ -3,6 +3,9 @@ const nextConfig = {
   experimental: {
     appDir: true,
   },
+  eslint: {
+    dirs: ["pages", "app", "components", "lib", "stories"],
```

Comment:
The stories dir wasn't part of the default set of directories Next.js ESLint integration uses: https://nextjs.org/docs/basic-features/eslint#linting-custom-directories-and-files